### ISSUE TITLE
Error occurred when executing KSampler:  No module named 'mac_specific'

### ISSUE THREAD

**Issue by 2-mq** [comfyanonymous/ComfyUI#1153]
Error occurred when executing KSampler:  No module named 'mac_specific'  File "/Volumes/T7Shield/ComfyUI/execution.py", line 151, in recursive_execute output_data, output_ui = get_output_data(obj, input_data_all) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/execution.py", line 81, in get_output_data return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/execution.py", line 74, in map_node_over_list results.append(getattr(obj, func)(**slice_dict(input_data_all, i))) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/nodes.py", line 1206, in sample return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/nodes.py", line 1167, in common_ksampler previewer = latent_preview.get_previewer(device, model.model.latent_format) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/latent_preview.py", line 65, in get_previewer taesd = TAESD(None, taesd_decoder_path).to(device) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/comfy/taesd/taesd.py", line 55, in __init__ self.decoder.load_state_dict(torch.load(decoder_path, map_location="cpu", weights_only=True)) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/custom_nodes/ComfyUI_tagger/modules/safe.py", line 106, in load return load_with_extra(filename, extra_handler=global_extra_handler, *args, **kwargs) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/Volumes/T7Shield/ComfyUI/custom_nodes/ComfyUI_tagger/modules/safe.py", line 131, in load_with_extra from . import shared File "/Volumes/T7Shield/ComfyUI/custom_nodes/ComfyUI_tagger/modules/shared.py", line 10, in  from . import errors,shared_items,devices File "/Volumes/T7Shield/ComfyUI/custom_nodes/ComfyUI_tagger/modules/devices.py", line 8, in  import mac_specific

Hi I'm trying to use control net on comfyUI but having trouble generating image with the error "No module named 'mac_specific'"

**Comment by createdeepstudio** [comment]
me too 
Error occurred when executing KSampler:

'ModuleList' object has no attribute '1'

File "D:\stable-diffusion\SDXL\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "D:\stable-diffusion\SDXL\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "D:\stable-diffusion\SDXL\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "D:\stable-diffusion\SDXL\ComfyUI\nodes.py", line 1206, in sample
return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise)
File "D:\stable-diffusion\SDXL\ComfyUI\nodes.py", line 1176, in common_ksampler
samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,
File "D:\stable-diffusion\SDXL\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\modules\impact\hacky.py", line 9, in informative_sample
return original_sample(*args, **kwargs)
File "D:\stable-diffusion\SDXL\ComfyUI\comfy\sample.py", line 93, in sample
samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)
File "D:\stable-diffusion\SDXL\ComfyUI\comfy\samplers.py", line 637, in sample
pre_run_control(self.model_wrap, negative + positive)
File "D:\stable-diffusion\SDXL\ComfyUI\comfy\samplers.py", line 481, in pre_run_control
x[1]['control'].pre_run(model.inner_model.inner_model, percent_to_timestep_function)
File "D:\stable-diffusion\SDXL\ComfyUI\comfy\sd.py", line 940, in pre_run
set_attr(self.control_model, k, self.control_weights[k].to(model_management.get_torch_device()))
File "D:\stable-diffusion\SDXL\ComfyUI\comfy\sd.py", line 241, in set_attr
obj = getattr(obj, name)
File "D:\stable-diffusion\SDXL\python_embeded\lib\site-packages\torch\nn\modules\module.py", line 1614, in __getattr__
raise AttributeError("'{}' object has no attribute '{}'".format(
